src/main/kotlin/com/terraformation/backend/tracking/db/PlantingSiteImporter.kt

```diff
@@ -156,7 +156,7 @@ class PlantingSiteImporter(
           val plantingSubzoneName = feature.properties[PLOT_NAME_PROPERTY]!!
```

Comment:
Dangit, just noticed this `PLOT_NAME_PROPERTY`. I'll update it when I work on the importer